server/src/tasks/schema.js

```diff
@@ -95,8 +97,16 @@ const taskResolvers = {
 function publish(actionType, data, pushClient) {
   if (pushClient) {
     pushClient.sender.send({
-      alert: `${actionType} ${data.title}`
-    }).then((response) => {
+      userData: {
+        title: data.title,
```

Comment:
should there be any string parsing here? excuse my lack of knowledge about push but what happens if we send a really long and ugly item? will the notification automatically add ellipses? 

So will `Thisisareallylongtexttodotitle` automatically become `Thisisareally....` ?

Reply:
I can answer this myself now after investigating. The notifications tool does parsing for us so ignore me 👍